pyproject.toml

```
@@ -20,6 +20,7 @@ dependencies = [
     "junitparser==2.8.0",
     "pydantic==1.10.2",
     "opensearch-py==2.0.0",
+    "slack_sdk==3.18.3"
```

```suggestion
    "slack_sdk==3.18.3",
```
to have fewer diff lines in the future